robotbase/src/main/java/frc/robot/commands/shooter/ShooterPickupFromSource.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.commands.shooter;
```

Comment:
This is fine for now. Eventually we'll have a command that sets the RPMs (I haven't done that yet). I'll fix this in my branch.